src/qibolab/compilers/compiler.py

```diff
@@ -129,12 +129,19 @@ def compile(self, circuit, platform):
 
         measurement_map = {}
         # process circuit gates
+        delays = defaultdict(int)
         for moment in circuit.queue.moments:
             moment_start = sequence.finish
             for gate in set(filter(lambda x: x is not None, moment)):
+                if isinstance(gate, gates.Align):
+                    for qubit in gate.qubits:
```

Comment:
The [qibo gate](https://github.com/qiboteam/qibo/blob/610765810100b26e6a1879a1444ae07b1107292a/src/qibo/gates/gates.py#L457) allows several qubits.

Reply:
Ok, it was just listed in the 1-qubit section:

https://qibo.science/qibo/latest/api-reference/qibo.html#align-a

But I guess it's just an interface to tell you want to apply many 1-qubit gates. The only thing I wonder is would you distinguish such "many 1-qubits" from a genuine multi-qubit gate, since I have the feeling I've already seen `if len(gate.qubits) == 1`.
Or we don't really need to ever distinguish the two cases?